fe/fe-core/src/main/java/com/starrocks/catalog/MaterializedView.java

```diff
@@ -188,6 +281,38 @@ public void gsonPostProcess() throws IOException {
             // it is checked when creation
             ((OlapTable) table).addRelatedMaterializedView(id);
         }
+        // analyze expression, because it converts to sql for serialize
+        ConnectContext connectContext = new ConnectContext();
+        connectContext.setCluster(SystemInfoService.DEFAULT_CLUSTER);
+        connectContext.setDatabase(db.getFullName());
+        // set privilege
+        connectContext.setQualifiedUser(Auth.ROOT_USER);
+        connectContext.setCurrentUserIdentity(UserIdentity.ROOT);
+        connectContext.setThreadLocalInfo();
```

Comment:
need to `setThreadLocalInfo` here?

Reply:
We have issue  Analyzer's method named `visitQueryStatement` which  i fixed it yesterday,  so we can remove this now.